rust-toolchain

```diff
@@ -1 +1 @@
-1.63.0
+nightly-2022-10-03
```

Comment:
Why do we need this? Can I request we use a stable release? I don't think this repository should use nightly at all.